sql/mysqld.cc

```diff
@@ -7006,9 +7010,22 @@ static int show_heartbeat_period(THD *thd, SHOW_VAR *var, char *buff,
   return 0;
 }
 
-
 #endif /* HAVE_REPLICATION */
 
+static int show_max_used_connections_time(THD *thd, SHOW_VAR *var, char *buff,
```

Comment:
Functions should be diveded with 2 spaces ( I think we need to update that in Coding standard) @LinuxJedi

Reply:
> Functions should be diveded with 2 spaces ( I think we need to update that in Coding standard) @LinuxJedi

It already explains that blocks should be indented with 2 spaces: https://github.com/MariaDB/server/blob/11.0/CODING_STANDARDS.md#indentation

Do you mean that functions should be _separated_ with 2 _blank lines_? If so, yeah, that should be added to CODING_STANDARDS.md.